mockgen/mockgen.go

```diff
@@ -411,3 +414,15 @@ func (g *generator) GenerateMockRecorderMethod(mockType string, m *model.Method)
 	g.p("}")
 	return nil
 }
+
+// Format gives a go-formatted copy of the buffer.
```

Comment:
let's call this Output.

// Output returns the generator's output, formatted in the standard Go style.